### PR TITLE
Rename CUSTOMER-DEFAULT userstore to DEFAULT

### PR DESCRIPTION
**Pull request by Rashmini**
### Purpose
$subject

### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/userstores/constants/user-store-constants.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export const DISABLED = "Disabled";
  * @constant
  * @type {string}
  */
-export const CONSUMER_USERSTORE = "CUSTOMER-DEFAULT";
+export const CONSUMER_USERSTORE = "DEFAULT";
```
Comment (inthirakumaaran) [inline apps/console/src/features/userstores/constants/user-store-constants.ts:130]: Why we are having this constant in Identity apps?

Comment (Rashmini) [reply apps/console/src/features/userstores/constants/user-store-constants.ts:130]: It should be removed from here. @DonOmalVindula is working on it.

**Review by inthirakumaaran — APPROVED** [review]

**Comment by brionmario** [comment]
@Rashmini What's the status of this PR?

**Comment by Rashmini** [comment]
This will be fixed with another PR.

**Pull request closed without merging**